app/models/traffic_cam.rb

```diff
@@ -1,2 +1,3 @@
 class TrafficCam < ApplicationRecord
+  belongs_to :location
```

Comment:
I could see this being its own PR, or updating the title/description of this PR to reflect that you're doing two different things.